date/date.php

```diff
@@ -528,7 +528,7 @@ function gmmktime(
  * </p>
  * @return bool true if the date given is valid; otherwise returns false.
  */
-#[Pure(true)]
+#[Pure]
```

Comment:
The [`checkdate()`](https://www.php.net/checkdate) function is independent of the current time and locale.